app/src/main/java/com/github/se/signify/ui/common/Practice.kt

```diff
@@ -0,0 +1,126 @@
+package com.github.se.signify.ui.common
+
+import android.util.Log
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.outlined.ArrowBack
+import androidx.compose.material.icons.automirrored.outlined.ArrowForward
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.github.se.signify.model.common.getIconResId
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+/**
+ * Composable function that displays a horizontally arranged letter navigator. Users can scroll
+ * through each letter in the alphabet using left and right arrow buttons, and click on a letter box
+ * to scroll to the corresponding item in a vertically scrollable list.
+ *
+ * @param coroutineScope The `CoroutineScope` for launching scroll actions.
+ * @param numbOfHeaders The number of headers at the top of the list, allowing for an offset when
+ *   scrolling to the selected letter.
+ */
+@Composable
+fun LetterDictionary(
```

Comment:
it feels a bit weird to have a button (a clickable that does nothing).
look next comment for better understanding to how to solve this typo.